catalog/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from catalog.apps import CatalogConfig
+from . import views
+
+app_name = CatalogConfig.name
+
+urlpatterns = [
+    path('', views.home, name='home'),
+    path('home.html', views.home, name='home'),
+    path('contacts.html', views.feedback, name='contacts'),
```

Comment:
Указываем не название шаблона, а путь, где страница будет доступна и обязательно слеш в конце
```suggestion
    path('contacts/', views.feedback, name='contacts'),
```
Не забудь поправить шаблоны